src/backend/auth/server.js

```
@@ -0,0 +1,94 @@
+
+dotenv.config();
+
+const app = express();
+app.use(cors({
+    origin: "http://localhost:5173", 
+    credentials: true
+}));
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+const CLIENT_ID = process.env.VITE_NCU_PORTAL_CLIENT_ID;
```

ESLint

[error] 'process' is not defined.

(no-undef)